fbchat/graphql.py

```diff
@@ -44,7 +44,7 @@ def get_customization_info(thread):
         rtn['nicknames'] = {}
         for k in info['participant_customizations']:
             rtn['nicknames'][k['participant_id']] = k.get('nickname')
-    else:
+    elif info['participant_customizations']:
```

Comment:
Yea so far I've only seen it come back as an empty array, I haven't seen the field not set yet, so I only wanted to handle that and didn't want to make any additional assumptions. But down for whatever you think is better!